src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -2630,40 +2467,26 @@ baddate:
         End Function
 
         Friend Function TryTokenAsContextualKeyword(t As SyntaxToken, ByRef k As KeywordSyntax) As Boolean
-            If t Is Nothing Then
-                Return False
-            End If
-
-            If t.Kind = SyntaxKind.IdentifierToken Then
-                Return TryIdentifierAsContextualKeyword(DirectCast(t, IdentifierTokenSyntax), k)
-            End If
-
+            If t Is Nothing Then Return False
```

Comment:
Please revert the formatting change here. I'm going to stop commenting on these, but please revert all the changes that are not relevant to shortening the method names.